lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -41,4 +50,16 @@ class PokemonOverviewPage extends StatelessWidget {
       ),
     );
   }
+
+  void _noPokemonAvailableDialog(BuildContext context, errorMessage) {
+    const snackBar = SnackBar(
+      content: Text(
+        snackBarErrorMessage,
+        style: TextStyle(color: Colors.red),
+      ),
+    );
+    WidgetsBinding.instance.addPostFrameCallback((_) {

```

Comment:
do you still need this addPostFrameCallback, seems not anymore since context is already available here as given from this build method 🙂